### PR TITLE
Fix typo: `onMessageRecieved` -> `onMessageReceived`

### PR DESCRIPTION
**Pull request by javiercr**
Fix spelling typo. The function is not exported so it shouldn't break anything.

### CODE REVIEW

**Review by canove — APPROVED** [review]

**Review by andresayac — APPROVED** [review]

**Review by ugurgungezerler — APPROVED** [review]

**Comment by javiercr** [comment]
This is just a one-line typo fix for a private function; it shouldn't have any external implications. Could we please get it merged? 🙏

**Review by PurpShell — APPROVED** [review]
sorry for the delays, I usually wait till we get 2 accepting reviews from @whiskeysockets members

**Pull request merged**
